ZUGFeRD-Test/ZUGFeRD20Tests.cs

```diff
@@ -531,6 +533,7 @@ public void TestWriteAndReadExtended()
             ms.Seek(0, SeekOrigin.Begin);
             InvoiceDescriptor loadedInvoice = InvoiceDescriptor.Load(ms);
 
+            Assert.AreEqual("A1", loadedInvoice.BusinessProcess);
```

Comment:
can you please create dedicated test cases instead of wiring it into the existing test cases?